stripe/utils.go

```diff
@@ -146,3 +149,15 @@ func CallSet(err ...error) (d diag.Diagnostics) {
 	}
 	return d
 }
+
+const maxOperations = 10
```

Comment:
If someone wants to reduce the number of parallels in the future, change this number. if someone wants to include wait sec, add it to the withRateLimiting func. GPT will tell you how to do it. 🤖